### DISCUSSION TITLE
How to update a item by items service without setting new update date

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#14748] [Archive]
Is there any way to prevent the autofields from overwriting the values which are set in the item. I want to copy a item in directus_activity and would be practical when the dates are the same like before?

**Comment by rokdd** [comment]
Okay I found a workaround.. so the only thing how I managed is to use [knex](http://knexjs.org/guide/raw.html#raw-parameter-binding) to write directly to the database (database object which is passed by the extension-sdk for API Endpoints):
```
database.update({date_updated:database.raw("?","2022-07-20T13:10:56.690Z")}).table(<your table>).where({ id:<your_id> })
```

**Reply by rijkvanzanten** [reply]
This would be the only way. If you call an update in Directus, all "update"  transformation handlers are called as well. There's currently no option to circumvent that
